templates/content/node--oe-sc-event--full.html.twig

```diff
@@ -45,10 +45,14 @@
 {% endset %}
 
 <article{{attributes}}>
-  {{ pattern('inpage_navigation', {
-    title: 'Page content',
-    links: inpage_navigation_links,
-    content: inpage_navigation_fields,
-    full_layout: true,
-  }) }}
+  {% if inpage_navigation_links is empty and description_items is not empty %}
```

Comment:
this is ugly and complex, would be better to wrap an if around the `set inpage_navigation_links`